modules/luci-base/luasrc/sys.lua

```diff
@@ -189,6 +189,23 @@ local function _nethints(what, callback)
 			end
 		end
 	)
+	
+	cur:foreach("dhcp", "odhcpd",
+		function(s)
+			if s.leasefile and fs.access(s.leasefile) then
+				for e in io.lines(s.leasefile) do
+					local iface, duid, iaid, name, ts, id, length, ip = e:match("^# (%S+) (%S+) (%S+) (%S+) (-?%d+) (%S+) (%S+) (.*)")
+					if ip and iaid == "ipv4" then
+						ip = ip:gsub("/[0-9]+","")
```

Comment:
I suggest to tweak the `e:match()` pattern above to not include `/[0-9]+` in the first place, e.g.
`e:match("^# (%S+) (%S+) (%S+) (%S+) (-?%d+) (%S+) (%S+) ([0-9a-f:.]+)/[0-9]+")`

Reply:
i removed the extra variables and moved duid iaid where the others locals are declared